uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Form/ManageEntitlements.php

```diff
@@ -335,8 +335,8 @@ private function exportCSV() {
         'employee_name' => $calculation->getContact()['display_name'],
         'leave_type' => $calculation->getAbsenceType()->title,
         'prev_year_entitlement' => $calculation->getPreviousPeriodProposedEntitlement(),
-        'days_taken' => $calculation->getNumberOfDaysTakenOnThePreviousPeriod(),
-        'remaining' => $calculation->getNumberOfDaysRemainingInThePreviousPeriod(),
+        'days_taken' => $calculation->getAmountUsedInPreviousPeriod(),
```

Comment:
I think we should rename this array key to `amount_taken`

Reply:
As this is more related to the screen itself, than to the Entitlement Calculation service, I decided to not touch it for now and then change it as part of PCHR-2675